### ISSUE TITLE
Идея - Awaking

### ISSUE THREAD

**Issue by hippogamesunity** [hippogamesunity/SpellMasters#63]
Появилась идея разблокировать дополнительного юнита в отряде при достижении определенных условий. Функция позволит разбавить геймплей на высоких уровнях. По выбору игрока раскрывается одна из 3-х пустых карт. Тип юнита заранее неизвестен, но генерируется в самом начале при создании отряда. Таким образом, получается кот в мешке)
![image](https://user-images.githubusercontent.com/7318172/31576929-3c0e8c26-b10e-11e7-9af2-8e3209bff29f.png)

**Comment by Ragewortt** [comment]
Т.е сначала генерируется весь отряд из 9 юнитов, а три из них разблокируются после достижения определенных условий?

**Comment by hippogamesunity** [comment]
Да, идея такая. Только пока 1 предлагаю открывать, чтобы не было 100% заполнение формации 3х3.
